modules/RTC/JitsiRemoteTrack.js

```diff
@@ -201,6 +201,13 @@ export default class JitsiRemoteTrack extends JitsiTrack {
         this.emit(JitsiTrackEvents.TRACK_VIDEOTYPE_CHANGED, type);
     }
 
+    /**
+     * Gets the content type of this video track.
+     */
+    getVideoType() {
```

Comment:
Let's define this in JitsiTrack. It could be used for local tracks too.

Reply:
unfortunately we don't have access to the videoType field in JitsiTrack but I can add a corresponding getter in JitsiLocalTrack?

Reply:
FTR, there is lots of code already accessing videoType directly.

Reply:
@gpolitis , Actually @saghul is correct!

Could you please remove the method and just use it directly! I don't think this is good practice but let's be consistent :) WDYT